html/assets/css/index.css

```
@@ -29,4 +29,8 @@
 .extension-view-project {
 	margin-left: 7px;
 	margin-right: 7px;
+}
+
+.tile {
```

Using this can have side effects on other tiles used on this page in the future. Could you elaborate another way to use it? Probably using a specific class along side tile, something like: 

```suggestion
.tile center {
```